src/components/AccountConnectionData.jsx

```diff
@@ -2,15 +2,16 @@ import styles from '../styles/accountConnectionData'
 
 import React from 'react'
 import { translate } from 'cozy-ui/react/I18n'
+import ClassNames from 'classnames'
 import ReactMarkdownWrapper from '../components/ReactMarkdownWrapper'
 import DataItem from '../components/DataItem'
 
-const AccountConnectionData = ({ t, connector }) => {
+const AccountConnectionData = ({ t, connector, hideMobile }) => {
```

Comment:
I feel that `hideMobile` is a little bit confusing, like a negative condition. What do you think if we replace it by `isMobile` which I feel clearer ?

Reply:
Hum actually I want to specify some parts that should be hidden on mobile which is different from knowing if we are on mobile or not :/

Reply:
What about `!isMobile` ? Or just inverting your ternary ?

Reply:
Ok I see your point now. 👍